python/lsst/fgcmcal/fgcmOutputProducts.py

```diff
@@ -295,10 +296,15 @@ def runDataRef(self, butler):
 
         # Check to make sure that the fgcmBuildStars config exists, to retrieve
         # the visit and ccd dataset tags
-        if not butler.datasetExists('fgcmBuildStars_config'):
-            raise RuntimeError("Cannot find fgcmBuildStars_config, which is prereq for fgcmOutputProducts")
+        if not butler.datasetExists('fgcmBuildStarsTable_config') and \
+                not butler.datasetExists('fgcmBuildStars_config'):
+            raise RuntimeError("Cannot find fgcmBuildStarsTable_config or fgcmBuildStars_config, "
+                               "which is prereq for fgcmOutputProducts")
 
-        fgcmBuildStarsConfig = butler.get('fgcmBuildStars_config')
+        if butler.datasetExists('fgcmBuildStarsTable_config'):
+            fgcmBuildStarsConfig = butler.get('fgcmBuildStarsTable_config')
+        else:
+            fgcmBuildStarsConfig = butler.get('fgcmBuildStars_config')
```

Comment:
I'm going to leave it since I don't know of a better way, but I understand the concern.

Reply:
I feel like this should be documented somewhere. Maybe a couple of lines in the Task .rst docs? Because a user of this might not immediately catch where the config came from.